extra/3-magic-8-ball.js

```diff
@@ -60,6 +85,69 @@ function shakeBall() {
 */
 function checkAnswer(answer) {
   //Write your code in here
+
```

Comment:
Veryyyy long if else  loop don't you think? Try to break down your array into 4 small ones. Then we can use them to compare if the inbound answer is inside one of the arrays. Also, this function doesn't correlate with your array at all.

Reply:
Wow! Maksim - this is so useful. Thank you for taking the time to review! I'm also very impressed that you fixed the bug in your version. I made a few corrections based on your advice. Hope I understood your feedback correctly.